cli_tools/gce_vm_image_import/importer/inflater.go

```diff
@@ -150,3 +152,46 @@ func (inflater *daisyInflater) cancel(reason string) bool {
 	inflater.wf.CancelWithReason(reason)
 	return true
 }
+
+func createDaisyVarsForFile(args ImportArguments,
+	fileInspector vdisk.VirtualDiskFileInspector, diskName string) map[string]string {
+	vars := map[string]string{
+		"source_disk_file": args.Source.Path(),
+		"import_network":   args.Network,
+		"import_subnet":    args.Subnet,
+		"disk_name":        diskName,
+	}
+
+	// To reduce the runtime permissions used on the inflation worker, we pre-allocate
+	// disks sufficient to hold the disk file and the inflated disk. If inspection fails,
+	// then the default values in the daisy workflow will be used. The scratch disk gets
+	// a padding factor to account for filesystem overhead.
+	deadline, cancelFunc := context.WithDeadline(context.Background(), time.Now().Add(inspectionTimeout))
+	defer cancelFunc()
+	metadata, err := fileInspector.Inspect(deadline, args.Source.Path())
+	if err == nil {
+		vars["inflated_disk_size_gb"] = fmt.Sprintf("%d", calculateInflatedSize(metadata))
+		vars["scratch_disk_size_gb"] = fmt.Sprintf("%d", calculateScratchDiskSize(metadata))
+	}
+	return vars
+}
+
+// Allocate extra room for filesystem overhead, and
+// ensure a minimum of 10GB (the minimum size of a GCP disk).
+func calculateScratchDiskSize(metadata vdisk.VirtualDiskFileMetadata) int64 {
+	// This uses the historic padding calculation from import_image.sh: add ten percent,
+	// and round up.
+	padded := int64(float64(metadata.PhysicalSizeGB)*1.1) + 1
+	if padded < 10 {
+		return 10
+	}
+	return padded
+}
+
+// Ensure a minimum of 10GB (the minimum size of a GCP disk)
+func calculateInflatedSize(metadata vdisk.VirtualDiskFileMetadata) int64 {
+	if metadata.VirtualSizeGB < 10 {
```

Comment:
10 should be extracted as a constant as it's used in multiple functions.

Reply:
sgtm